### DISCUSSION TITLE
How to download ckan.exe

### DISCUSSION THREAD

**Discussion by bobby2411** [KSP-CKAN/CKAN#3278] [Q&A] [answered]
Where is the application file?!? I am using Windows 10, and I need to know where the file to open the app is!

**Comment by HebaruSan** [comment]
You copy of ckan.exe should be wherever your browser saved it after downloading.

**Reply by bobby2411** [reply]
well, where would it be stored normally at?

**Reply by bobby2411** [reply]
cuz i can not find it at all

**Reply by DasSkelett** [reply]
Your user's `Download` folder, maybe? Usually your browser prompts you about where you want to save a file.
If you really can't find it anymore, just download it again, but this time remember where you save it.

**Reply by bobby2411** [reply]
i am already in the downloads/ckan folder

**Reply by bobby2411** [reply]
and i have checked every subfolder there is.

**Reply by bobby2411** [reply]
![image](https://user-images.githubusercontent.com/77946045/105651630-07d11d00-5e7d-11eb-8c04-db34fcf7cbd4.png)
where would it be?

**Reply by HebaruSan** [reply]
If you can't find files you've downloaded, that's more of a question for your browser vendor. The CKAN team is not responsible for whatever browser you're using.

**Reply by HebaruSan** [reply]
Oh, you downloaded the wrong file. Get the `ckan.exe`, not the source.

**Reply by DasSkelett** [reply]
Looks like you downloaded CKAN's source code. You likely don't want that.
Go to the release section, and download `ckan.exe`, under "Assets":
https://github.com/KSP-CKAN/CKAN/releases/latest

**Reply by HebaruSan** [reply]
@DasSkelett would you move that to a separate reply so I can mark it as the answer? This could help other folks who are confused by GitHub.

**Reply by bobby2411** [reply]
but i am not using a browser, I'm using the built-in file explorer.

**Reply by bobby2411** [reply]
OOF.

**Comment by DasSkelett** [comment, marked as answer]
Looks like you downloaded CKAN's source code. You likely don't want that.

Go to the Release section, and download `ckan.exe`, under "Assets":
https://github.com/KSP-CKAN/CKAN/releases/latest

See also the guide for installing on Windows:
https://github.com/KSP-CKAN/CKAN/wiki/Installing-CKAN-on-Windows

**Reply by bobby2411** [reply]
TYSM(Thank You So Much)
